misc/rot13.c

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file rot13.c
```

Comment:
No need to add the filename. It's best to let Doxygen automatically detect it, otherwise, you'd have to specify the whole directory as well.
```suggestion
 * @file
```